buildscripts/checkdeps.sh

```diff
@@ -25,6 +25,11 @@ _init() {
     OSX_VERSION="10.8"
     KNAME=$(uname -s)
     ARCH=$(uname -m)
```

Comment:
does `uname -m` exist in Solaris? does `isainfo -k` provide different value over `uname -m`?